README.md

```diff
@@ -46,10 +46,11 @@ end
 ```
 The second to last line defines the circle geometry using a signed distance function. The `AutoBody` function uses [automatic differentiation](https://github.com/JuliaDiff/) to infer the other geometric parameter automatically. Replace the circle's distance function with any other, and now you have the flow around something else... such as a [donut](examples/ThreeD_donut.jl), a [block](examples/TwoD_block.jl) or the [Julia logo](examples/TwoD_Julia.jl). Finally, the last line defines the `Simulation` by passing in the `dims=(n+2,m+2)` and the other parameters we've defined.
 
-Now we can create a simulation (first line) and run it forward in time (second line)
+Now we can create a simulation (first line) and run it forward in time (second and third line)
```

Comment:
good catch, but it is only the last (now the third) line that runs the simulation forward in time.